server/src-lib/Hasura/RQL/DML/Select/Types.hs

```diff
@@ -316,6 +317,24 @@ data FunctionArgsExpG a
 emptyFunctionArgsExp :: FunctionArgsExpG a
 emptyFunctionArgsExp = FunctionArgsExp [] HM.empty
 
+-- If argument positional index is less than or equal to
+-- length of 'positional' arguments then insert the value in
+-- 'positional' arguments else insert the value with
+-- argument name in 'named' arguments
```

Comment:
This comment needs to start with a `|` so it can get picked up by Haddock. Also, since [our style guide currently encourages a 100-character line length](https://github.com/hasura/graphql-engine/blob/950d3842e4662c2ff5d8d5b20ac64bf9f9387b80/server/CONTRIBUTING.md#line-length), could you wrap this comment to 100 character lines?